packages/frontend/src/pages/Settings.tsx

```diff
@@ -105,7 +105,7 @@ const Settings: FC = () => {
                         )}
                         {orgData &&
                             !orgData.needsProject &&
-                            user.data?.ability?.can('manage', 'Project') && (
+                            user.data?.ability?.can('view', 'Project') && (
```

Comment:
why this change ?

Reply:
So member users can see the list of all the projects they have access to and access the project settings page. I instead have hidden the create and delete button if they don't have permissions to delete it.